src/gleam/string.gleam

```diff
@@ -148,27 +203,34 @@ pub fn contains(does haystack: String, contain needle: String) -> Bool {
     erl_contains(haystack, needle)
 }
 
-// TODO
-// TODO: Not sure about the name and labels here
 /// See if the second string starts with the first one.
 ///
 /// ## Examples
 /// ```gleam
 /// starts_with(does: "theory", start_with: "ory") == False
 /// ```
 ///
-// pub fn starts_with(does string: String, start_with prefix: String) -> String {}
+external fn erl_string_starts_with(string: String, prefix: String) -> Bool =
+  "gleam_stdlib" "string_starts_with"
```

Comment:
We can export this function directly rather than wrapping it with another.

Reply:
The parser doesn't allow named parameters on external functions, see the below comment.

Reply:
Sweet good to know! changed!